packages/server/graphql/public/mutations/deleteUsers.ts

```diff
@@ -0,0 +1,101 @@
+import {USER_EMAIL_LIMIT} from '../../../postgres/constants'
+import {getUsersByEmails} from '../../../postgres/queries/getUsersByEmails'
+import {getUserById} from '../../../postgres/queries/getUsersByIds'
+import {
+  getUserId,
+  isSuperUser,
+  isUserBillingLeader,
+  isUserOrgAdmin
+} from '../../../utils/authorization'
+import {GQLContext} from '../../graphql'
+import {markUserSoftDeleted} from '../../mutations/deleteUser'
+import softDeleteUser from '../../mutations/helpers/softDeleteUser'
+import {MutationResolvers} from '../resolverTypes'
+
+const deleteUsers: MutationResolvers['deleteUsers'] = async (
+  _source,
+  {emails}: {emails: string[]},
+  {authToken, dataLoader}: GQLContext
+) => {
+  if (emails.length === 0) {
+    return {error: {message: 'Provide emails'}}
+  }
+
+  if (emails.length > USER_EMAIL_LIMIT) {
+    return {error: {message: 'Cannot delete more than 100 users at once'}}
```

Comment:
+1 would be nice if the 100 in the error message would be replaced with the limit variable as well.